### PR TITLE
[8.0] backport l10n_ch_qr_bill_scan

### PR DESCRIPTION
**Pull request by fredzamoabg**
backporting from https://github.com/OCA/l10n-switzerland/pull/561

### CODE REVIEW

File: l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py [hunk 1]
```diff
@@ -152,7 +195,7 @@ def test_scan_QR_QRR(self):
         self.assertEqual(invoice.reference, QRR)
         self.assertEqual(invoice.state, "draft")
         iban = invoice.partner_bank_id.acc_number
-        self.assertEqual(iban, QR_IBAN.replace(" ",""))
+        self.assertEqual(iban.replace(" ", ""), QR_IBAN.replace(" ", ""))
```
Comment (SimoneVagile) [inline l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py:198]: As above

File: l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py [hunk 2]
```diff
@@ -164,7 +207,7 @@ def test_scan_QR_CF(self):
         self.assertEqual(invoice.reference, CF)
         self.assertEqual(invoice.state, "draft")
         iban = invoice.partner_bank_id.acc_number
-        self.assertEqual(iban, CH_IBAN.replace(' ', ''))
+        self.assertEqual(iban.replace(' ', ''), CH_IBAN.replace(' ', ''))
```
Comment (SimoneVagile) [inline l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py:210]: As above

File: l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py [hunk 3]
```diff
@@ -200,7 +243,7 @@ def test_scan_QR_extra_first_lines(self):
         self.assertFalse(invoice.reference)
         self.assertEqual(invoice.state, "draft")
         iban = invoice.partner_bank_id.acc_number
-        self.assertEqual(iban, CH_IBAN.replace(' ', ''))
+        self.assertEqual(iban.replace(' ', ''), CH_IBAN.replace(' ', ''))
```
Comment (SimoneVagile) [inline l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py:246]: As above

File: l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py [hunk 4]
```diff
@@ -139,7 +182,7 @@ def test_scan_QR_free_ref(self):
...
```
[the remark this answers is not in the document]
Comment (fredzamoabg) [reply l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py:185]: don't you like replace method?

**Review by SimoneVagile — APPROVED** [review]

**Pull request closed without merging**